### DISCUSSION TITLE
ModelX model to production

### DISCUSSION THREAD

**Discussion by tmasuku28** [fumitoh/modelx#102] [General]
Hi there

I am relatively new to python, and I've just used modelx and the lifelib to build a model similar to the BasicTerm_M. Now, we are aiming to transition this to production. So, I to have some form of graphical user interface that will make it easier for non coders to use the model. is this possible for such models that are built with modelx considering we are running on everything on iPython? If so, what are the steps that I could follow? 

Thanks! :)

**Comment by fumitoh** [comment]
By 'graphical user interface,' are you referring to a GUI for developing models or for executing them? If you're asking about development, the Spyder IDE with the spyder-modelx plugin is a great option. It provides a user-friendly interface for model development.

**Reply by tmasuku28** [reply]
I mean for executing the models

**Reply by fumitoh** [reply]
Running is as easy as writing a short script:

```python
import modelx as mx
mx.read_model(r"C:\path\to\your\model").Projection.result_pv().to_excel("result.xlsx")
```
Save it as `run.py` and from the command line prompt, run:
```
python run.py
```
Or save the code above as *.BAT file and double click it to run.

**Reply by tmasuku28** [reply]
Perfect! This helps a lot!

Thank you so much @fumitoh! Really love what you've built!
